network/pound/pound.SlackBuild

```diff
@@ -109,8 +109,8 @@ cp -a \
 cat $CWD/$PRGNAM.SlackBuild > $PKG/usr/doc/$PRGNAM-$VERSION/$PRGNAM.SlackBuild
 
 mkdir -p $PKG/etc $PKG/etc/rc.d
-cp $CWD/pound.cfg $PKG/etc/pound.cfg.new
-cp $CWD/rc.pound $PKG/etc/rc.d/rc.pound.new
+install -D -m 0644 -o root -g root $CWD/pound.cfg $PKG/etc/pound.cfg.new
+install -D -m 0755 -o root -g root $CWD/rc.pound $PKG/etc/rc.d/rc.pound.new
```

Comment:
shouldn't rc.xxx be installed with 0644 by default? they need to enable (+x) it manually if it's needed

Reply:
I grepped all the slackbuilds and it seems some do 0644 some 0755. Is there an FAQ for which is "correct"?

Reply:
i think 755 should only be binaries, but rc.xxx should be 644 by default

Reply:
I switched it to 644